frontend/src/components/nodes/NodeRow.tsx

```diff
@@ -20,8 +25,28 @@ export default class extends React.PureComponent<Props> {
     ["BodySync", "Syncing body"],
     ["NoSync", ""],
   ]);
+
+  state: State = { blockDiff: 0 };
+
+  componentDidMount() {
+    new BlocksApi().getLatestBlockHeight().then((lastBlockHeight) =>
+      this.setState({
+        blockDiff: Math.abs(
+          lastBlockHeight.lastBlockHeight - this.props.node.lastHeight
```

Comment:
what is this for?

Reply:
https://github.com/near/near-explorer/issues/305

Reply:
There is no point in saving it to state, it is actually more expensive to save it than to compute. Also, this latest block height should be fetched only once, and passed via React Context, so we don't query it for every single node. Also, make sure that it takes into account the auto-refresh logic.